src/vm/assembly.cpp

```diff
@@ -2349,49 +2249,6 @@ void DECLSPEC_NORETURN Assembly::ThrowBadImageException(LPCUTF8 pszNameSpace,
 
 
 #ifdef FEATURE_COMINTEROP
-//
-// Manage an ITypeLib pointer for this Assembly.
-//
-ITypeLib* Assembly::GetTypeLib()
-{
-    CONTRACTL
-    {
-        NOTHROW;
-        GC_TRIGGERS;
-        FORBID_FAULT;
-    }
-    CONTRACTL_END
-
-    // Get the value we are going to return.
-    ITypeLib *pResult = m_pITypeLib;
-    // If there is a value, AddRef() it.
-    if (pResult && pResult != (ITypeLib*)-1)
-        pResult->AddRef();
-    return pResult;
-} // ITypeLib* Assembly::GetTypeLib()
-
-void Assembly::SetTypeLib(ITypeLib *pNew)
-{
-    CONTRACTL
-    {
-        NOTHROW;
-        GC_TRIGGERS;
-        FORBID_FAULT;
-    }
-    CONTRACTL_END
-
-    ITypeLib *pOld;
-    pOld = InterlockedExchangeT(&m_pITypeLib, pNew);
-    // TypeLibs are refcounted pointers.
-    if (pNew != pOld)
-    {
-        if (pNew && pNew != (ITypeLib*)-1)
-            pNew->AddRef();
-        if (pOld && pOld != (ITypeLib*)-1)
-            pOld->Release();
-    }
-} // void Assembly::SetTypeLib()
-
```

Comment:
@AaronRobinsonMSFT  Do we need these for the COM interop support around IDispatch that you've been doing?